service/src/test/java/com/ge/predix/controller/test/PolicyEvalWithGraphDbControllerIT.java

```diff
@@ -202,6 +276,30 @@ public void testPolicyEvaluation(final Zone zone, final PolicySet testPolicySet,
                 Effect.PERMIT };
     }
 
+    /**
+     * Test that evaluation is successful when the resource and/or subject attributes exceed the length. The policy set will return
+     * indeterminate because the traversal limit is exceeded.
+     */
+    Object[] evaluationWithResourceAttributesExceedingTraversalLimitData() {
+        String errorMessage = "The number of attributes on this resource "
```

Comment:
"The number of attributes on resource '<resource id>' exceeds the maximum of '<limit>'"